src/components/BorrowerProfile/LendCta.vue

```diff
@@ -0,0 +1,358 @@
+<template>
+	<span class="tw-z-1">
+		<div class="tw-fixed
+					tw-left-0
+					md:tw-relative
+					md:tw-left-unset
+					tw-bottom-0
+					tw-right-0
+					md:tw-rounded-b
+					md:tw-rounded-t-none
+					lg:tw-rounded-t
+					tw-bg-white
+					tw-border
+					tw-bt-1
+					tw-border-gray-300
+					md:tw-border-none
+					tw-pt-1
+					tw-pl-2.5
+					tw-pr-2.5
+					lg:tw-pl-4
+					lg:tw-pr-4"
+		>
+			<p class="tw-text-h3 tw-pt-3 lg:tw-mb-3 tw-hidden lg:tw-inline-block">
+				{{ lgScreenheadline }}
+			</p>
+			<span class="tw-flex tw-pb-1 lg:tw-pb-3">
+				<kv-ui-select
+					v-if="hideShowLendDropdown"
+					id="LoanAmountDropdown"
+					class="tw-pr-2.5 tw--mb-2"
+					v-model="selectedOption"
+				>
+					<option
+						v-for="price in prices"
```

Comment:
There's already a prop named `price` in this component, so this should be changed to something different, i.e. `priceOption in prices`